packages/web-app-files/src/views/spaces/Project.vue

```diff
@@ -1,36 +1,112 @@
 <template>
   <div class="space-overview">
-    <list-loader v-if="loadFilesListTask.isRunning" />
+    <list-loader v-if="loadResourcesTask.isRunning" />
     <template v-else>
       <not-found-message v-if="!space.id" class="space-not-found oc-height-1-1" />
-      <div
-        v-else-if="isSpaceRoot"
-        class="oc-grid oc-grid-match oc-px-m oc-mt-m"
-        :class="{ 'oc-child-width-1-1@s': imageExpanded, 'oc-child-width-1-3@s': !imageExpanded }"
-      >
-        <div v-if="imageContent">
-          <img
-            :class="{ expanded: imageExpanded }"
-            class="space-overview-image oc-cursor-pointer"
-            alt=""
-            :src="'data:image/jpeg;base64,' + imageContent"
-            @click="toggleImageExpanded"
-          />
-        </div>
-        <div>
-          <h1 class="space-overview-name oc-mb-s">{{ space.name }}</h1>
-          <p v-if="space.description" class="oc-mt-rm">{{ space.description }}</p>
+      <div v-else-if="isSpaceRoot">
+        <portal v-if="$data.$_editReadmeContent_modalOpen" to="app.runtime.modal">
+          <oc-modal
```

Comment:
Feels weird to have exactly the same modal code here and in the `FileActions.vue`. Was it intended or didn't work out to deduplicate the code? Would like to hear some context. 😃

Reply:
Was not quite sure if we even keep the modal because of all the back and forth, so wasn't worth to me to create an extra component, will do this with the next PR.